napari/layers/labels/_labels_mouse_bindings.py

```diff
@@ -35,3 +35,27 @@ def fill(layer, event):
     """Fill in an area with the currently selected label."""
     # on press
     layer.fill(layer.coordinates, layer._value, layer.selected_label)
+
+
+def erase(layer, event):
```

Comment:
I worry this functions adds a lot of code duplication with the paint method for little gained - the only different is now we use `layer.selected_label` instead of `layer.background_label`. I think allowing the `selected_label` to be the `background_label` is fine, the background label is a label after all, just a special one, so I think I'd prefer this to be dropped and inside the ERASE mode setter just set the `selected_label` to be the `background_label`. I'll leave a comment there too